### PR TITLE
ES 861360 Modify the syntax in LaTeX

### PR DESCRIPTION
**Pull request by Venkateshmuruganandam**
HI All
Modify the syntax in LaTeX.

Regards
Venkatesh

### CODE REVIEW

File: File-Formats/DocIO/Working-with-LaTeX.md [hunk 1]
```diff
@@ -320,7 +320,7 @@ using WordDocument document = new WordDocument();
 document.EnsureMinimal();
 
 //Append an border box equation using LaTeX.
-document.LastParagraph.AppendMath(@"\boxed{x^2 + y^2 = z^2}");
+document.LastParagraph.AppendMath(@"\boxed{{x}^2+{y}^2={z}^2}");
```
Comment (MathanKumarVaradhaRajaPerumal) [inline File-Formats/DocIO/Working-with-LaTeX.md:323]: Add curly braces for 2 also.

File: File-Formats/DocIO/Working-with-LaTeX.md [hunk 2]
```diff
@@ -1440,7 +1441,7 @@ using WordDocument document = new WordDocument();
 document.EnsureMinimal();
 
 //Append an Left SubSuperScript equation using LaTeX.
-document.LastParagraph.AppendMath(@"{_40}^{20}}100");
+document.LastParagraph.AppendMath(@"{_40^{20}}100");
```
Comment (MathanKumarVaradhaRajaPerumal) [inline File-Formats/DocIO/Working-with-LaTeX.md:1444]: Please modify properly it is invalid syntax

**Review by MathanKumarVaradhaRajaPerumal — CHANGES_REQUESTED** [review]

File: File-Formats/DocIO/Working-with-LaTeX.md [hunk 3]
```diff
@@ -165,7 +165,7 @@ using WordDocument document = new WordDocument();
 document.EnsureMinimal();
 
 //Append an bar equation using LaTeX.
-document.LastParagraph.AppendMath(@"\dot{a}");
+document.LastParagraph.AppendMath(@"\overline{a}");
```
Comment (MathanKumarVaradhaRajaPerumal) [inline File-Formats/DocIO/Working-with-LaTeX.md:168]: Why the same changes not done for Windows specific and vb

File: File-Formats/DocIO/Working-with-LaTeX.md [hunk 4]
```diff
@@ -1335,7 +1331,7 @@ The following table demonstrates the LaTeX equivalent to professional format rad
 <tr>
 <td>1.</td>
 <td><img src="WorkingwithMathematicalEquation_images/Radical1.png" alt="Radical equation"></td>
-<td>\sqrt a</td>
+<td>@"\sqrt{a}"</td>
```
Comment (MathanKumarVaradhaRajaPerumal) [inline File-Formats/DocIO/Working-with-LaTeX.md:1334]: Whether the @ symbol is mandatory?

File: File-Formats/DocIO/Working-with-LaTeX.md [hunk 5]
```diff
@@ -1440,7 +1436,7 @@ using WordDocument document = new WordDocument();
 document.EnsureMinimal();
 
 //Append an Left SubSuperScript equation using LaTeX.
-document.LastParagraph.AppendMath(@"{_40}^{20}}100");
+document.LastParagraph.AppendMath(@"{_{40}^{20}}100");
```
Comment (MathanKumarVaradhaRajaPerumal) [inline File-Formats/DocIO/Working-with-LaTeX.md:1439]: Add curly braces for 100 too

File: File-Formats/DocIO/Working-with-LaTeX.md [hunk 6]
```diff
@@ -1547,7 +1543,7 @@ Dim document As WordDocument = New WordDocument()
 document.EnsureMinimal()
 
 'Append an Right SubSuperScript equation using LaTeX.
-document.LastParagraph.AppendMath(@”{100}_{40}^{20}")
+document.LastParagraph.AppendMath(@"{100}_{40}^{20}")
```
Comment (MathanKumarVaradhaRajaPerumal) [inline File-Formats/DocIO/Working-with-LaTeX.md:1546]: Why the same changes not done for Windows specific and vb

**Review by MathanKumarVaradhaRajaPerumal — CHANGES_REQUESTED** [review]

**Review by MathanKumarVaradhaRajaPerumal — APPROVED** [review]

**Review by MohanaselvamJothi — APPROVED** [review]

**Pull request merged**